### DISCUSSION TITLE
Chapter Titles

### DISCUSSION THREAD

**Discussion by saarstar15** [sultim-t/xash-rt#96] [Q&A]
Hey, is it possible to restore the original chapter titles by editing the files?

I noticed they still play after I quicksaved during a chapter transition.

Thanks!

**Comment by sultim-t** [comment]
Oof, I haven't provided any switch for this...
You may delete the chapter logo .ktx2 files, but the chapter text won't be shown at all in that case

**Comment by saarstar15** [comment]
No worries! Definitely not a priority at all, haha.

I think I may just edit the .ktx2 files in the meantime. 

Also wanted to thank you for making this incredible project! 110% my new favourite way to play Half-life.
